src/server/ocpp/utils/OCPPUtils.ts

```diff
@@ -1583,10 +1587,12 @@ export default class OCPPUtils {
         tenantID: tenant.id,
         action: ServerAction.UPDATE_CHARGING_STATION_WITH_TEMPLATE,
         module: MODULE_NAME, method: 'enrichChargingStationWithTemplate',
-        message: 'No Template has been found for this Charging Station',
-        detailedMessages: { chargingStation }
+        message: 'No template has been found',
+        detailedMessages: {
+          chargingStationData: LoggingHelper.shrinkChargingStationProperties(chargingStation)
+        }
       });
-      chargingStation.manualConfiguration = true;
+      chargingStation.manualConfiguration = true; // To be clarified! - Why is this changed here???
```

Comment:
For the BootNotification ! We save the charging station just after calling this function...
Smells like a hack :)